packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -331,15 +329,27 @@ const TooltipContentImpl = React.forwardRef<TooltipContentImplElement, TooltipCo
 
     useEscapeKeydown(() => onClose());
 
+    // Close this tooltip if another one opens
     React.useEffect(() => {
-      // Close this tooltip if another one opens
       document.addEventListener(TOOLTIP_OPEN, onClose);
       return () => document.removeEventListener(TOOLTIP_OPEN, onClose);
     }, [onClose]);
 
+    // Close the tooltip if the trigger is scrolled
+    const { trigger } = context;
```

Comment:
Shall we avoid destructuring here? we don't tend to do so unless we're accessing context methods in our effects.